### DISCUSSION TITLE
What is the best possible way to use github copilot ?

### DISCUSSION THREAD

**Discussion by RavinduDasanayaka** [community/community#139785] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

I'm started to use co pilot and how can i use it efficiently ?

**Comment by Ravinduchathuranga** [comment, marked as answer]
this depended on your workflow! this is a supportive tool to improve your coding ability and speed to push in to next level!

**Reply by RavinduDasanayaka** [reply]
thank you !
